cloudio-dev-environment/docker-compose.yml

```diff
@@ -21,3 +21,25 @@ services:
     image: mongo:latest
     ports:
     - "27017:27017"
+    
+  cloudio-service:
```

Comment:
This docker-compose file (or the whole folder) is meant to get an environment (services) up for development, so it does not make sense to have the services run themselves in that environment. 

I propose to create a new repository with a name like "cloudio-deploy-examples" and copy the folder as "docker-compose" into that folder.

Reply:
make sense to me, I can do that. I'll remove that from here

Reply:
Good. I am already doing the example. So if it is done you can test it and maybe change things that do not work...

Reply:
Alright, good to me.
4fdd736 does revert the change to docker compose